resources/views/employees/form.blade.php

```diff
@@ -0,0 +1,62 @@
+<form action="@isset($employee) {{ route('employees.update', $employee->id) }} @else {{ route('employees.store') }} @endisset" class="form-validate is-alter" id="employee_form">
+    @isset($employee)
+        @method('PUT')
+    @endisset
+        @csrf
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    @isset($employee)
+                    <h5 class="modal-title">Edit Employee</h5>
+                    @else
+                    <h5 class="modal-title">Add Employee</h5>
+                    @endisset
+                    <a href="#" class="close" data-dismiss="modal" aria-label="Close">
+                        <em class="icon ni ni-cross"></em>
+                    </a>
+                </div>
+                <div class="modal-body" id="employee_form_div">
+            <div class="form-group">
+            <label class="form-label required" for="full-name">First Name</label>
+            <div class="form-control-wrap">
+                <input type="text" class="form-control" value="@isset($employee){{ $employee->first_name }}@endisset" name="first_name" required>
+            </div>
+        </div>
+        <div class="form-group">
+            <label class="form-label required" for="email-address">Last Name</label>
+            <div class="form-control-wrap">
+                <input type="text" class="form-control" id="last_name" value="@isset($employee){{ $employee->last_name }}@endisset" name="last_name" required>
+            </div>
+        </div>
+        <div class="form-group">
+            <label class="form-label required" for="phone-no">Email</label>
```

Comment:
Id for input and for attribute of label must be same

Reply:
Done.